plumbing/object/change_adaptor_test.go

```diff
@@ -1,11 +1,10 @@
-package difftree
+package object
```

Comment:
because change_adaptor_test.go is really huge, y keept the change_adaptor.go file with his original content.